app/javascript/application.js

```diff
@@ -4,12 +4,25 @@ import {DataTable} from "simple-datatables"
 import "@hotwired/turbo-rails"
 import "./controllers"
 
-document.addEventListener("DOMContentLoaded", function() {
-  const dataTable = new DataTable("#collection-table", {
-    paging: false,
-    labels: {
-      placeholder: 'Filter...',
-    },
-    columns: [{ select: 0, sort: 'asc' }]
-  });
+document.addEventListener("turbo:frame-render", function() {
```

Comment:
This JS needed some changes to (1) work with turbo-frames; (2) only fire when the element was present; (3) add a new datatable for registration jobs.